presentation/src/main/java/com/cointrend/presentation/ui/about/AboutScreen.kt

```diff
@@ -0,0 +1,236 @@
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AboutScreen(
+) {
+    val focusManager = LocalFocusManager.current
+
+    Scaffold(
+        modifier = Modifier,
+        topBar = {
+            TopAppBar(
+                title = {IconButton(onClick = {
+                    System.out.println("Go Back")
+                }) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_arrow_back_ios),
+                        contentDescription = "Return to previous screen",
+                        modifier = Modifier.padding(start = 12.dp)
+                    )
+                }
+                }
+            )
+        }
+    ) { padding ->
+        Column(
+            Modifier
+                .fillMaxSize()
+                .padding(padding)
+                .pointerInput(Unit) {
+                    detectTapGestures(
+                        onTap = { focusManager.clearFocus() },
+                        onPress = { focusManager.clearFocus() },
+                        onLongPress = { focusManager.clearFocus() },
+                        onDoubleTap = { focusManager.clearFocus() }
+                    )
+                }
+        ) {
+            LazyColumn() {
+                // The Logp
+                item {
+                    CoinTrend()
+                }
+
+                // Setion title 16dp SOURCE CODE
+                item {
+                    SectionTitle(title = "Source Code", modifier = Modifier.padding(defaultHorizontalPadding))
+                }
+
+                item {
+                    Column(
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .background(
+                                color = StocksDarkBackgroundTranslucent,
+                                shape = MaterialTheme.shapes.large
+                            ),
+                    ) {
+                        SectionInfoItemTest(name = "Github", info = "https://github.com/CoinTrend/CoinTrend", image = R.drawable.ic_github, showDivider = false)
+                    }
+                }
+                item {
+                    Spacer(modifier = Modifier.size(16.dp))
+                }
+                // Setion title 16dp CONTACT
+                item {
+                    SectionTitle(title = "Contact", modifier = Modifier.padding(defaultHorizontalPadding))
+                }
+                item {
+                    Column(
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .background(
+                                color = StocksDarkBackgroundTranslucent,
+                                shape = MaterialTheme.shapes.large
+                            ),
+                    ) {
+                        SectionInfoItemTest(name = "Email", info = "cointrend.info@gmail.com", image = R.drawable.ic_gmail, showDivider = true)
```

Comment:
Try to find a better email icon. Maybe there is one in the Material library that I used across the App like `Icons.Default.TrendingUp`